samples-aspnetcore-3x/self-hosted-login/okta-aspnetcore-mvc-example.sln

```diff
@@ -5,6 +5,12 @@ VisualStudioVersion = 16.0.29519.87
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "okta-aspnetcore-mvc-example", "okta-aspnetcore-mvc-example\okta-aspnetcore-mvc-example.csproj", "{916FD7FA-5E6B-452C-B34C-BB2DCBB258CF}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Okta.Idx.Sdk", "..\..\submodules\okta-idx-dotnet\src\Okta.Idx.Sdk\Okta.Idx.Sdk.csproj", "{D288EE9A-2029-4C43-B137-0981D7C19E16}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Idx", "Idx", "{F6DAEEFF-6CF9-48FB-AA45-06CF7E01FCA7}"
```

Comment:
Do we need this project for the sample?

Reply:
That project is actually a Visual Studio folder, I intended to use it for submodules that could be removed later when we reference the binaries.  Trivial to remove if it's undesirable.